runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java

```diff
@@ -399,6 +397,8 @@ public void layoutContainer(Container parent)
 			frame.toFront();
 			requestFocus();
 			giveClientFocus();
+
+			trayIcon = SwingUtil.createTrayIcon(ICON, properties.getTitle(), frame);
```

Comment:
This will probably cause the Notifier to be initialized with null tray icon, so tray notifications will stop working

Reply:
I checked just to be sure, and it doesn't. Started up the game, logged in, attacked a cow, got an idle notification after it died.

The only time `getTrayIcon` is called is in `sendTrayNotification`, and it doesn't store the result. It just ignores the potential notification if it returns `null`, and it's unlikely to anyways as AFAIK RuneLite doesn't send any notifications on startup.

Reply:
Ah, my mistake, I forgot that there was change in the ClientUI and notifier to make them created by guice, so yea it should be fine now.